install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2630,6 +2630,46 @@ clustermesh:
         #     cpu: 100m
         #     memory: 100Mi
 
+    kvstoremesh:
```

Comment:
The `kvstoremesh` configuration options are currently placed at this level for consistency with `etcd`, but I'm wondering whether they should be moved one level up. WDYT?